docker/tools/Dockerfile

```diff
@@ -1,3 +1,49 @@
+# Build Kopia binary
+FROM golang:1.19-bullseye AS builder
+
+ARG kopiaBuildCommit
+ARG kopiaRepoOrg
```

Comment:
I think best practice is to use `snake_case` instead of camelCase for ARG and ENV

Reply:
If arg is not set, does it fail the build?

Reply:
We do expect the arg to be always set.

Reply:
Ankit: it is expected to be set when called via `goreleaser`. What happens if someone tries to build the image directly? what happens if the argument is unset? do things fails?

Reply:
I have kept default values in Dockerfile, allowing them to be passed using build-arg if required.